### PR TITLE
Fixes for indel realignment workflow

### PR DESCRIPTION
**Pull request by murphycj2**
I needed to make these changes before I was able to successfully test the indel realignment sub-workflow (here is the [PR](https://github.com/msk-access/cwl_subworkflows/pull/150)).

I am still a novice to to cwl/toil, so let me know if I misunderstood anything.

### CODE REVIEW

File: abra2_2.22/abra2_2.22.cwl [hunk 1]
```diff
@@ -44,7 +44,7 @@ inputs:
       prefix: '--ref'
     doc: Genome reference location
     secondaryFiles:
-      - ^.fai
+      - ^.fasta.fai
```
Comment (rhshah) [inline abra2_2.22/abra2_2.22.cwl:47]: This depends on the name of the reference file you are using, if the name is `hg19.fasta` and the `hg19.fasta.fai` is fai then `^.fai` should work.

Comment (rhshah) [reply abra2_2.22/abra2_2.22.cwl:47]: @murphycj2  please let me know if that is not the case

**Review by rhshah — APPROVED** [review]

File: abra2_2.22/abra2_2.22.cwl (hunk 1 above)
Comment (murphycj2) [reply abra2_2.22/abra2_2.22.cwl:47]: @rhshah  that is what i thought, but using just `^.fai` did not work for me. Happened before to me. i am no expert in toil/cwl, so I am not sure what is going on.

Comment (ionox0) [reply abra2_2.22/abra2_2.22.cwl:47]: I think the ^ means "remove the last part of the suffix after '.'" 

So probably just ".fai" should work

Comment (murphycj2) [reply abra2_2.22/abra2_2.22.cwl]: removing the ^ works, thanks @ionox0 !

**Pull request merged**